.env_example

```diff
@@ -0,0 +1,6 @@
+PORT=""
+DATABASE=""
+INIT_INVITE=""
+JWT_SECRET=""
+DISCORD_ID=""
+DISCORD_SECRET=""
```

Comment:
```suggestion
PORT=5000
WSPORT=5001
DATABASE=/srv/database.sqlite
INIT_INVITE=URMOM
JWT_SECRET=THISISASECRET
DISCORD_ID=APP_ID_HERE
DISCORD_SECRET=APP_SECRET_HERE
```

Reply:
This illustrates default values and doesn't promote the usage of quotes which might potentially break stuff